wazo_dird_client/commands/helpers/__init__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from xivo_lib_rest_client import RESTCommand
+
+
+class DirdRESTCommand(RESTCommand):
```

Comment:
I don't now if it would be best to put this code in a separate file and import it. What is your opinion on this?

Reply:
Yeah, I got lazy. fixing it